.github/workflows/rust.yml

```diff
@@ -159,7 +162,7 @@ jobs:
         | xargs -0 scripts/target/release/check_copyright_header
     - name: Put lint toolchain file in place
       run: |
-        ln -sf toolchains/lint/rust-toolchain.toml
+        ln -sf toolchains/nightly/rust-toolchain.toml
```

Comment:
Is this line still needed at all then?

Reply:
Yes, we're still using the `stable` toolchain by default.